### DISCUSSION TITLE
Unable to log into new account

### DISCUSSION THREAD

**Discussion by AlexCrest56** [vercel/community#3861] [Account Issue] [answered]
### Question

Hi, I have signed up with a hobby account, but the system won't let me log in.  I receive this message each time:   We are unable to complete your signup. Please read this guide for further details.  Does anyone have any advice? My email is alexbrianhiscock@gmail.com  thanks, Alex

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
